manual/contents/src/operations/installation-deployment/file-locations.asciidoc

```diff
@@ -6,12 +6,24 @@ Many of these paths are configurable; see <<configuration-settings>> for details
 
 [options="header"]
 |===
-|Package               |Configuration                                  |Data
-|Linux or OS X tarball |_<neo4j-root>/conf/neo4j.conf_                 |_<neo4j-root>/data_
-|Windows zip           |_<neo4j-root>\conf\neo4j.conf_                 |_<neo4j-root>\data_
-|Debian/Ubuntu .deb    |_/etc/neo4j/neo4j.conf_                        |_/var/lib/neo4j_
-|Windows desktop       |_%APPDATA%\Neo4j Community Edition\neo4j.conf_ |_%APPDATA%\Neo4j Community Edition_
-|OS X desktop          |_$\{HOME\}/Documents/Neo4j/neo4j.conf_         |_$\{HOME\}/Documents/Neo4j_
+|Package               |Configuration                                  |Data                                |Logs
+|Linux or OS X tarball |_<neo4j-root>/conf/neo4j.conf_                 |_<neo4j-root>/data_                 |_<neo4j-root>/logs_
+|Windows zip           |_<neo4j-root>\conf\neo4j.conf_                 |_<neo4j-root>\data_                 |_<neo4j-root>/logs_
+|Debian/Ubuntu .deb    |_/etc/neo4j/neo4j.conf_                        |_/var/lib/neo4j_                    |_/var/log/neo4j_
+|Windows desktop       |_%APPDATA%\Neo4j Community Edition\neo4j.conf_ |_%APPDATA%\Neo4j Community Edition_ |_%APPDATA%\Neo4j Community Edition\logs_
+|OS X desktop          |_$\{HOME\}/Documents/Neo4j/neo4j.conf_         |_$\{HOME\}/Documents/Neo4j_         |_$\{HOME\}/Documents/Neo4j/logs_
```

Comment:
Have you verified the Windows and OS X paths or guessed? You might be surprised -- I have been.